cmd/estuary-shuttle/main.go

```diff
@@ -2442,3 +2446,15 @@ func (s *Shuttle) handleGetSystemConfig(e echo.Context) error {
 	}
 	return e.JSON(http.StatusOK, resp)
 }
+
+func (s *Shuttle) handleStorageProviderList(e echo.Context) error {
+	n, err := strconv.Atoi(e.Param("n"))
+
+	if err != nil {
+		return err
+	}
+
+	resp := s.PPM.Result.GetTopPeers(n)
```

Comment:
right now, this might return an empty list if it's not populated yet, but will be addressed in #679

Reply:
We should evaluate if we can just cache this as well or not.